### ISSUE TITLE
Add alt attributes to all event images

### ISSUE THREAD

**Issue by SohamJuneja** [SohamJuneja/OpenSourceDSC#11]
All event images in index.html are missing alt attributes, making them inaccessible to screen readers and failing accessibility standards.

**Comment by ramanbansal1** [comment]
I can do this. Can you assign this to me.

**Comment by ramanbansal1** [comment]
I can do this. Can you assign this to me.
